x-pack/test/functional/services/observability/alerts/common.ts

```diff
@@ -307,5 +317,6 @@ export function ObservabilityAlertsCommonProvider({
     viewRuleDetailsButtonClick,
     viewRuleDetailsLinkClick,
     getAlertsFlyoutViewRuleDetailsLinkOrFail,
+    getAlertStatValue,
```

Comment:
@claudiopro same here. I would call it `getRuleStateValue`

Reply:
@ersin-erdal is working on a renaming wrong references of alert to rule for infra and rule_registry plugins, so we could create a subticket to the bigger Meta issue he will create to rename alerts to rules for the `observability` plugin as well.